app/views/queue_items/index.html.haml

```diff
@@ -4,29 +4,31 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            - @queue_items.each do |queue_item|
+        = bootstrap_form_tag url: "/update_queue" do 
+          %table.table
+            %thead
               %tr
-                %td
-                  = queue_item.position
-                %td
-                  = link_to queue_item.video_title, queue_item.video
-                %td
-                  = button_to "Play", nil, class: "btn btn-default"
-                %td
-                  = queue_item.rating
-                %td
-                  = link_to queue_item.category_name, queue_item.category
-                %td
-                  = link_to queue_item, method: :delete do
-                    %i.glyphicon.glyphicon-remove
-        = button_to "Update Instant Queue", nil, class: "btn btn-default"
+                %th(width="10%") List Order
+                %th(width="30%") Video Title
+                %th(width="10%") Play
+                %th(width="20%") Rating
+                %th(width="15%") Genre
+                %th(width="15%") Remove
+            %tbody
+              - @queue_items.each do |queue_item|
+                %tr
+                  %td
+                    = hidden_field_tag "queue_items[][id]", queue_item.id
+                    = number_field_tag "queue_items[][position]", queue_item.position, class: "form-control"
+                  %td
+                    = link_to queue_item.video_title, queue_item.video
+                  %td
+                    = button_to "Play", nil, class: "btn btn-default"
+                  %td
+                    = select_tag "queue_items[][rating]", options_for_select([5,4,3,2,1].map {|value| ["#{value} Stars", value]},queue_item.rating), include_blank: true,  class: "form-control"

```

Comment:
True, in fact I have used before but this time just didnt realize. Also created the helper. I Suppose that now that i´m using the tests and not paying the same attention to the web interface. I suppose that with capybara this will change.
Thanks.